Application/Contact/Mail/Service/Data.php

```diff
@@ -195,13 +195,14 @@ public function getMailAllByCompany(TblCompany $tblCompany)
 
     /**
      * @param TblPerson $tblPerson
-     * @param TblMail   $tblMail
-     * @param TblType   $tblType
-     * @param string    $Remark
+     * @param TblMail $tblMail
+     * @param TblType $tblType
+     * @param string $Remark
+     * @param bool $IsAccountUserAlias
      *
      * @return TblToPerson
      */
-    public function addMailToPerson(TblPerson $tblPerson, TblMail $tblMail, TblType $tblType, $Remark)
+    public function addMailToPerson(TblPerson $tblPerson, TblMail $tblMail, TblType $tblType, $Remark, $IsAccountUserAlias = false)
```

Comment:
Codacy found an issue: [Incorrect spacing between argument "$IsAccountUserAlias" and equals sign; expected 0 but found 1](https://app.codacy.com/manual/gerdchristian-kunze/SPHERE-Framework/pullRequest?prid=5757982)